enhancements/coreos-encrypted-by-default.md

```diff
@@ -0,0 +1,425 @@
+---
+title: CoreOS Encrypted Disks By Default
+authors:
+  - "Ben Howard @behoward"
+reviewers:
+  - "Colin Walters @cgwalters"
+  - "Steve Miler @ashrow"
+  - "Ian McLeod @imcleod"
+  - "Micah Abbot @miabott"
+approvers:
+  - TBD
+creation-date: 2019-09-20
+last-updated: 2019-09-20
+status: provisional
+---
+
+# CoreOS-Encrypted-by-Default
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift/docs]
+- [ ] Transition Plan defined for Ignition handling of LUKS root volumes
+
+## Summary
+
+Openshift clusters will be encrypted at the operating system level by default. CoreOS boot images starting with 4.3 will be encrypted using a null-cipher. At boot time, the instance will be re-encrypted using AES-256-CBC and will not require user-intervention for boot and reboot.
```

Comment:
Let's try to split up as much as possible the user facing aspects ("encryption by default") from the *implementation* ("null cipher on boot").

Reply:
Updated.